index.js

```diff
@@ -0,0 +1,75 @@
+'use strict'
+
+const fs = require('fs');
+const os = require('os')
+
+fs.readFile('./files/pair-programming.txt',(err, data) =>{if(err) throw err;
+  let arr = data;
+  let newarr = [];
```

Comment:
newarr is a little hard to read, try camel case newArr or resultsArray or just results